### PR TITLE
ppsspp: Add version 1.12.3

### PR DESCRIPTION
**Pull request by beyondmeat**

### CODE REVIEW

**Comment by beyondmeat** [comment]
ready?

**Comment by UnicorNora** [comment]
This package seems fine @Calinou

File: bucket/ppsspp.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\")) {",
+        "   New-item \"$persist_dir\" -ItemType Directory | Out-Null",
+        "}"
+    ],
+    "bin": "PPSSPPWindows64.exe",
+    "shortcuts": [
+        [
+            "PPSSPPWindows64.exe",
+            "PPSSPP,"
```
Comment (Calinou) [inline bucket/ppsspp.json:17]:
```suggestion
            "PPSSPP"
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**